### ISSUE TITLE
Show _what_ props change when in debug-mode

### ISSUE THREAD

**Issue by torgeir** [omniscientjs/omniscient#132]
It would be useful if the `.debug()` options on omniscient and shouldComponentUpdate outputted _what_ props changed, instead of just `=> true (props have changed)`. 

This would ease fixing stuff like unnescessary renders that happen because of changed passed function references, and help show when you'd benefit from something like [omnipotent](https://github.com/omniscientjs/omnipotent)'s ignore

**Comment by mikaelbr** [comment]
So you are thinking diffing the two states/props? That could work, but should only happen when it has changed.

**Comment by torgeir** [comment]
Yep, that's where debug log already occurs. Diffing or not, it would be useful to pinpoint the prop that changed, so it's thrown right in your face 👊  We already know something changed, we might as well say what, so there's no need for digging around..

**Comment by mikaelbr** [comment]
Agreed.
